src/components/integrated/OakLessonBottomNav/OakLessonBottomNav.tsx

```diff
@@ -50,10 +50,7 @@ export const OakQuizBottomNav = ({
   return (
     <StyledOakFlex
       $flexDirection={["column", "row"]}
-      $background={["bg-primary", "bg-decorative1-very-subdued"]}
       $pa="inner-padding-m"
-      $borderColor="border-inverted"
-      $bt="border-solid-xl"
       $minHeight="all-spacing-9"
```

Comment:
Leaving it up to the layout to decide the background colour which makes it easier to manage.